src/core/iomgr/pollset_multipoller_with_epoll.c

```diff
@@ -180,7 +181,9 @@ static void multipoll_with_epoll_pollset_maybe_work(
   pfds[1].events = POLLIN;
   pfds[1].revents = 0;
 
+  GRPC_IOMGR_START_BLOCKING_REGION;
```

Comment:
Thanks, I will address those. It's probably a distracting or harmful false
positive to mark something if it never actually blocks, so I'll scrap these.

On Thu, Sep 24, 2015, 5:37 PM David Klempner <notifications@github.com>
wrote:

> In src/core/iomgr/pollset_multipoller_with_epoll.c
> <https://github.com/grpc/grpc/pull/3476#discussion_r40390282>:
>
> > @@ -180,7 +181,9 @@ static void multipoll_with_epoll_pollset_maybe_work(
> >    pfds[1].events = POLLIN;
> >    pfds[1].revents = 0;
> >
> > +  GRPC_IOMGR_START_BLOCKING_REGION;
>
> It may be worthwhile (although probably not in the initial version of this
> PR) to do an unguarded 0 timeout poll here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/3476/files#r40390282>.
>